### ISSUE TITLE
Height limit locked to 70 blocks

### ISSUE THREAD

**Issue by PongoSapiens** [Ephys/minecraft-magic-feather#20]
Minecraft 1.18.2
Forge 40.1.68
Magic Feather 4.2.0

It appears that no matter how you configure the vertical range limit, you cannot rise more than 70 blocks above the nearest Beacon, and if you are set to "FullHeight", you cannot fly more than 70 blocks below the nearest beacon. Looking at the code, I suspect that that is because the POI code is limited to a 70 block search radius. In other words if it could find the Beacon it would let you ascend to the highest possible block, but as it only searches +/- 70 blocks for the Beacon, this limit takes affect first.

**Comment by ephys** [comment]
Yes that's very likely the reason. I think I'll have to return to using the loaded tile entity list instead of a POI to fix this

**Comment by aholster** [comment]
would it be possible to backport this change to the 1.18.2 version?
